freighter-index/sql/publish/get-or-insert-crate.sql

```diff
@@ -1,7 +1,4 @@
-with ins as (insert into crates (name) values ($1) on conflict do nothing returning *)
```

Comment:
It was such a pointlessly elegant and performant sql statement. I'll be sad to see it go.